test/common/formatter/substitution_formatter_speed_test.cc

```diff
@@ -63,6 +81,47 @@ static void BM_AccessLogFormatter(benchmark::State& state) {
 }
 BENCHMARK(BM_AccessLogFormatter);
 
+// NOLINTNEXTLINE(readability-identifier-naming)
+static void BM_StructAccessLogFormatter(benchmark::State& state) {
+  std::unique_ptr<Envoy::TestStreamInfo> stream_info = makeStreamInfo();
+  std::unique_ptr<Envoy::Formatter::StructFormatter> struct_formatter = makeStructFormatter(false);
+
+  size_t output_bytes = 0;
+  Http::TestRequestHeaderMapImpl request_headers;
+  Http::TestResponseHeaderMapImpl response_headers;
+  Http::TestResponseTrailerMapImpl response_trailers;
+  std::string body;
+  for (auto _ : state) {
```

Comment:
This (and line 115) triggers a clang-tidy error, but I just copied it from other tests in this file.

Reply:
`for (auto _ : state) { // NOLINT` is how this has been solved elsewhere. Clang-tidy is only applied to changes in our CI, which is why others in this file didn't trigger it. You may have to fix those as well.